.github/workflows/test.yml

```diff
@@ -37,6 +46,15 @@ jobs:
         with:
           config: .testcoverage.yml
 
+  unit:
```

Comment:
optional - I believe `make test` includes some integration tests and only excludes the e2e tests. It may be worth either renaming this from "unit" to something more generic or making a new testing job for integration tests.

Reply:
This is a good observation. Will switch back.